packages/@uppy/core/src/index.js

```diff
@@ -1135,6 +1135,14 @@ class Uppy {
         isPaused: false
       })
 
+      // Remote providers sometimes don't tell us the file size,
+      // but we can know how many bytes we uploaded once the upload is complete.
+      if (file.size == null) {
+        this.setFileState(file.id, {
+          size: currentProgress.bytesTotal || uploadResp.bytesUploaded
```

Comment:
I think there is a change that `currentProgress.bytesTotal` is not going to be the actual size of the file because some events can be lost due to throttling, I think that I should always use `uploadResp.bytesUploaded`. I could look like

```javascript
if (uploadResp.bytesUploaded) {
 ...
 size: uploadResp.bytesUploaded
 ...
}
```